tequilapi/endpoints/identities.go

```diff
@@ -646,27 +683,24 @@ func AddRoutesForIdentities(
 		bprovider:         bprovider,
 		addressStorage:    addressStorage,
 	}
-	router.GET("/identities", idAPI.List)
-	router.POST("/identities", idAPI.Create)
-	router.PUT("/identities/:id", func(resp http.ResponseWriter, request *http.Request, params httprouter.Params) {
-		// TODO: remove this hack when we replace our router
-		switch params.ByName("id") {
-		case "current":
-			idAPI.Current(resp, request, params)
-		default:
-			http.NotFound(resp, request)
+	return func(e *gin.Engine) error {
+		identityGroup := e.Group("/identities")
+		{
+			identityGroup.GET("", idAPI.List)
+			identityGroup.POST("", idAPI.Create)
+			identityGroup.PUT("/current", idAPI.Current)
+			identityGroup.GET("/:id", idAPI.Get)
+			identityGroup.GET("/:id/status", idAPI.Get)
+			identityGroup.PUT("/:id/unlock", idAPI.Unlock)
+			identityGroup.GET("/:id/registration", idAPI.RegistrationStatus)
+			identityGroup.GET("/:id/beneficiary", idAPI.Beneficiary)
+			identityGroup.GET("/:id/referral", idAPI.GetReferralToken)
+			identityGroup.GET("/:id/referral-available", idAPI.ReferralTokenAvailable)
+			identityGroup.GET("/:id/payout-address", idAPI.GetPayoutAddress)
+			identityGroup.PUT("/:id/payout-address", idAPI.SavePayoutAddress)
+			identityGroup.PUT("/identities/:id/balance/refresh", idAPI.BalanceRefresh)
```

Comment:
Group is `identities` so this produces something like `/identities/identities/:id/balance/refresh`. Probably missed when merged master.